### ISSUE TITLE
[English]: Container Runtime Interface (CRI)

### ISSUE THREAD

**Issue by TarekMSayed** [cncf/glossary#534]
### Details (optional)

Optional details

**Comment by jaynarayan20** [comment]
i want to contribute to this issue can i take this?

**Comment by JasonMorgan** [comment]
Ok so we have [this](https://kubernetes.io/docs/concepts/architecture/cri/) definition in the k8s docs. @TarekMSayed can you take a look and see if you still think the glossary needs an independent definition? 

Thanks!

**Comment by TarekMSayed** [comment]
Yes, @JasonMorgan I think so since the definition in k8s docs is not generic enough it is very related to Kubernetes and how Kublete interacts with the CRI but does not explain it clearly. Also, I believe it does not suit the non-tech reader requirements.
Thanks

**Comment by CathPag** [comment]
The term should not be Kubernetes specific but rather explain the term more generally.

**Comment by iamNoah1** [comment]
@JasonMorgan friendly ping ;)

**Comment by aj11anuj** [comment]
Hello, can i work on this issue?

**Comment by iamNoah1** [comment]
Hei @aj11anuj, sure. Please make sure tough that you read the [contribution guidelines](https://glossary.cncf.io/contribute/) and come say hi on slack.

**Comment by iamNoah1** [comment]
Hei @aj11anuj, friendly ping to see if you are still interested in working on this issue?!

**Comment by aj11anuj** [comment]
Uh-oh 😅, I almost forgot about this issue. I will surely complete it by the end of this week.
Thanks for reminding me :)
